overgang (#76)
thesis20/report: sections/experiments/datasets.tex

\begin{align*}
    NDCG = \frac{1.93}{2.13} = 0.91
\end{align*}

> Now that the metrics have been examined, the method employed for splitting the data is interesting to investigate.

Reviewer: ```suggestion
Now that the metrics have been examined, we will look into splitting the data.
```
